### DISCUSSION TITLE
Custom MAC adress on ESP32 board

### DISCUSSION THREAD

**Discussion by Eaed007** [darthcloud/BlueRetro#872] [General]
Hello there, I'm wanting to use the ESP32 board on my Wii Gamecube ports to use my PS3 controllers on it. I realized that is a pain in the butt having to set the MAC adress manually intro the joysticks each time I pair with the PS3 and then back to the Wii. I thought there may be a way to Match the ESP board Bluetooth MAC adress to My ps3 Bluetooth MAC adress. I found some Arduino post with información how to change the MAC adress on the ESP board but i'm not to familiar with code programing, so i'm not sure if it's possible to add the línes yo change the MAC adress into the Gamecube code for blueretro. Any ideas or inputs Will be greatly appreciatted.

I,ll be adding a swicth to turno off and on ESP board on the Wii to avoid the controllers to pair yo then incorrect console while playing.

**Comment by darthcloud** [comment]
It's possible to put a bdaddr.bin named file on the ESP32 spiffs and it will use that MAC that is inside that binary file.

But doing so is not simple and I don't have documentation for it.

One day I will add support to upload file though the web interface, until then let's say it's not possible.
